esx_service/cli/vmdkops_admin.py

```diff
@@ -280,6 +280,22 @@ def commands():
                             }
                         },
 
+                        'replace': {
```

Comment:
nit: this should have been done as a separate PR - it is not related to VMODL and could have been merged separately and earlier.